packages/transformers/vue/src/VueTransformer.js

```diff
@@ -320,12 +320,12 @@ ${
             style.lang = extname(style.src);
           }
           switch (style.lang) {
-            case 'less':
-            case 'stylus':
-            case 'styl':
-            case 'scss':
-            case 'sass':
-            case 'css':
+            case '.less':
```

Comment:
can you instead trim of the `.` so it still matches without the dot

Reply:
Ok